devicemodel/hw/platform/acpi/acpi_parser.c

```diff
@@ -0,0 +1,129 @@
+/*-
+ * Copyright (C) 2021 Intel Corporation. All rights reserved.
```

Comment:
```suggestion
 * Copyright (C) 2022 Intel Corporation.
```

Reply:
Was that file originally from FreeBSD? In other words, is it appropriate to completely remove the original Copyright notice that was in here?

Reply:
In the beginning, the whole acpi parser file is imported, but in the end only one function(local_parse_madt) is kept.
Should the copyright of FreeBSD be announced?

Reply:
Original copyright must remain but you can add a second copyright line.

Reply:
Updated